src/app.rs

```diff
@@ -81,6 +82,11 @@ impl App {
                     script_var_process::on_application_death();
                     std::process::exit(0);
                 }
+                EwwCommand::CloseAll => {
+                    log::info!("Received close command, closing all windows");
+                    self.script_var_handler.stop_all();
+                    self.windows.drain().for_each(|(_, w)| w.close());
```

Comment:
please use `self.close_window` instead, as seen in `EwwCommand::CloseWindow` below